cmd/aries-js-worker/main.go

```diff
@@ -93,7 +94,7 @@ func main() {
 		ready <- struct{}{}
 	}
 
-	select {}
+	time.Sleep(math.MaxInt64)
```

Comment:
if it works, it works. An explanation would be awesome though...

Reply:
Hope it helps https://groups.google.com/forum/#!topic/golang-nuts/rtUIY5_m51I

Reply:
Not really.

I'm ok with trying out this change but only if you put a comment with some explanation of the problem we're trying to solve and be clear that it is MAGIC and we don't really know how it works.

Reply:
Some quick links:

* https://stackoverflow.com/a/32148983/1623885
* https://github.com/golang/go/blob/master/src/runtime/time.go#L47
* https://golang.org/src/runtime/select.go#L118
* https://golang.org/src/runtime/proc.go#L287